<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallScreen</class>
 <widget class="QDialog" name="CallScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background: #ddd;
	color: #333;
}

QComboBox, QPushButton, QListWidget {
	border: 1px solid gray;
	border-radius: 5px;
}

QListWidget {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #F6F6F6, stop: 1.0 #EFEFEF);
}

QComboBox {
    padding: 1px 3px 1px 3px;
}

 QComboBox, QComboBox::drop-down:editable {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                  stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
 }

QComboBox:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}

QComboBox:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ccc);
}

 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;
     border-width: 0px;
 }

QComboBox::down-arrow {
     image: url(res/down.png);
 }

QComboBox QAbstractItemView:editable::item:selected {
     alternate-background-color: gray;
 }

QPushButton {
	padding: 2px;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                  stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}

QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ccc);
}</string>
  </property>
  <widget class="Player" name="playerRemote" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="Player" name="playerLocal" native="true">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonEndCall">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>470</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	padding: 2px;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E10000, stop: 0.4 #DD0000,
                                  stop: 0.5 #D80000, stop: 1.0 #D30000);
	color: #ddd
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #f00, stop: 1 #b00);
}

QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #f00, stop: 1 #c00);
}</string>
   </property>
   <property name="text">
    <string>End call</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboCamera">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>150</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>130</y>
     <width>61</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Webcam:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Microphone:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboMicrophone">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>200</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Player</class>
   <extends>QWidget</extends>
   <header>player.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
